sympy/plotting/plot.py

```diff
@@ -1062,8 +1064,8 @@ def __init__(self, parent):
                 self.ax[i].spines['right'].set_color('none')
                 self.ax[i].spines['bottom'].set_position('zero')
                 self.ax[i].spines['top'].set_color('none')
-                self.ax[i].spines['left'].set_smart_bounds(True)
-                self.ax[i].spines['bottom'].set_smart_bounds(False)
+                # self.ax[i].spines['left'].set_smart_bounds(True)
+                # self.ax[i].spines['bottom'].set_smart_bounds(False)
```

Comment:
Why are these lines commented out? If they are not needed any more they should be removed.

Reply:
These two lines are deprecated in Matplotlib 3.2 and will be removed in later releases. And this set_smart_bounds method is also causing issue #18247.
I was confused about whether to remove it or not.